### ISSUE TITLE
Xunit removes special chars from suite name

### ISSUE THREAD

**Issue by mfurtado-mdsol** [lukejpreston/xunit-viewer#5]
I see a problem with the formatting of suiteName applied by Xunit. If you compare the `suiteName` property and the suite title in green background, you will see the special chars like `@, ., _, :` are removed. 

![screen shot 2017-05-31 at 17 39 34](https://cloud.githubusercontent.com/assets/6390925/26643235/91fe1c2e-4628-11e7-96ef-26d4f4d48168.png)

Is there a way to override this behavior? Thanks much.

**Comment by lukejpreston** [comment]
can you supply some sample data please?

**Comment by mfurtado-mdsol** [comment]
Hey @lukejpreston, below is the output produced by `wdio-junit-reporter`.

```
<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="release2017_1_0_mcc272151_10_view_listings_page" timestamp="2017-05-31T15:25:03" time="2.723" tests="6" failures="0" errors="0" skipped="0">
    <properties>
      <property name="specId" value="82b2938fd94d9f4454379788268e8e9e"/>
      <property name="suiteName" value="@release2017.1.0, @MCC272151_10: View listings page"/>
      <property name="capabilities" value="chrome"/>
      <property name="file" value="./test/features/visit_listing_by_context.feature"/>
    </properties>
    <testcase classname="chrome.release2017_1_0_mcc272151_10_view_listings_page" name="i_logged_into_an_application" time="12.079"/>
    <testcase classname="chrome.release2017_1_0_mcc272151_10_view_listings_page" name="i_navigate_to_the_listings_page" time="10.876"/>
    <testcase classname="chrome.release2017_1_0_mcc272151_10_view_listings_page" name="i_can_see_tabular_list_of_records" time="10.008"/>
  </testsuite>
</testsuites>
```

For following scenario:
```
  @release2017.1.0 @MCC272151_10
  Scenario: View listings page
    Given I logged into an application
    When I navigate to the listings page
    Then I can see tabular list of records
```

**Comment by mfurtado-mdsol** [comment]
Hey @lukejpreston, is it fine to replace `change-case` with `[titlecase](https://www.npmjs.com/package/titlecase)` to fix this issue?

**Comment by lukejpreston** [comment]
this is now published 5.0.4
